app/assets/typescript/index.ts

```diff
@@ -157,6 +171,10 @@ function addButton(text: string, command: string, {hexes, tracks, boosters}: {he
     button.attr("data-boosters", JSON.stringify(boosters));
   }
 
+  if (leech) {
```

Comment:
There's no need for the `leech` parameters. Parameters are used for two-steps command (eg click 'build a mine', and select corresponding hex). Accepting/declining leech is a one-step command